test/screenshot/README.md

```diff
@@ -1,22 +1,160 @@
 # Screenshot Tests
 
-## Local dev server
+Prevent visual regressions by running screenshot tests on every PR.
 
-Run:
+## Quick start
+
+### API credentials
+
+Add the following to your `~/.bash_profile` or `~/.bashrc` file:
+
+```bash
+export CBT_USERNAME='you@example.com'
+export CBT_AUTHKEY='example'
+export MDC_GCLOUD_SERVICE_ACCOUNT_KEY_FILE_PATH='path/to/gcp-credentials.json'
+```
+
+Credentials can be found here:
+
+* [CrossBrowserTesting.com > Account](https://crossbrowsertesting.com/account) \
+    `Authkey` is listed under the `User Profile` section
+* [Google Cloud Console > IAM & admin > Service accounts](https://console.cloud.google.com/iam-admin/serviceaccounts?project=material-components-web) \
+    Click the `︙` icon on the right side of the service account, then choose `Create key` 
+
+### Test your changes
+
+Modify a Sass file:
+
+```bash
+echo '.mdc-button:not(:disabled){color:red}' >> packages/mdc-button/mdc-button.scss
+```
+ 
+Run the tests:
+
+```bash
+npm run screenshot:test
+```
+
+You should see something like this in the terminal:
+
+```
+Found 65 screenshot diffs!
+https://storage.googleapis.com/mdc-web-screenshot-tests/advorak/2018/05/22/16_23_58_618/af3d7271f/report.html
+```
+
+## Basic usage
+
+### Updating "golden" screenshots
+
+```bash
+npm run screenshot:update-goldens
+```
+
+In addition to running the tests, this command will also update your local `test/screenshot/golden.json` file with the 
+newly captured screenshots.
+
+### Rerunning a subset of tests
+
+You can rerun a subset of the tests without running the entire suite, filtering by browser and/or URL:
+
+```bash
+npm run screenshot:update-goldens -- \
+  --mdc-include-url=mdc-button/classes/dense \
+  --mdc-include-browser=ie@11
+```
+
+**IMPORTANT:** Note the `--` between `screenshot:test` and `--help`. This is required by `npm`.
```

Comment:
This doesn't match the invocation immediately above. Maybe genericize this remark to "between the script name and its arguments"

Reply:
Done